pnba.py

```diff
@@ -144,4 +148,11 @@ def send_message(self, recipient, message):
             level="info",
         )
         logger.info(publish_alert)
+        store_publication(
+            country_code="Unknown",
+            platform_name=self.platform,
+            source="Platforms",
+            gateway_client="Unknown",
+            status=status,
+    )
```

Comment:
We should pass the `published` value directly here instead of using the `status` variable.